CMakeLists.txt

```diff
@@ -87,13 +74,27 @@ if(APPLE)
     endif()
     message(STATUS "Using MoltenVK repo location at ${MOLTENVK_REPO_ROOT}")
 endif()
-message(STATUS "Using find_package to locate Vulkan")
-find_package(Vulkan)
+
 find_package(VulkanHeaders REQUIRED CONFIG)
-get_filename_component(Vulkan_LIBRARY_DIR ${Vulkan_LIBRARY} DIRECTORY)
-message(STATUS "Vulkan FOUND = ${Vulkan_FOUND}")
-message(STATUS "Vulkan Lib Dir = ${Vulkan_LIBRARY_DIR}")
-message(STATUS "Vulkan Lib = ${Vulkan_LIBRARY}")
+
+# Find and create the Vulkan::Vulkan library using the VULKAN_LOADER_INSTALL_DIR for the vulkan loader and VULKAN_HEADERS_INSTALL_DIR for the include
+if(WIN32)
+    find_library(Vulkan_LIBRARY
+        NAMES vulkan-1
+        HINTS
+            "${VULKAN_LOADER_INSTALL_DIR}/lib"
+            "${VULKAN_LOADER_INSTALL_DIR}/bin"
```

Comment:
Which library are we looking for? The static or dynamic library?

Reply:
Ostensibly the static library, but thats only for compilation. To *run* the tests, the dynamic library is needed, and will have to be copied over to the tests folder. This is not done currently, and was an omission on my part. (everything worked on my dev machine cause I have the loader installed to system32).